Lib/asyncio/tasks.py

```diff
@@ -469,11 +466,18 @@ def _wait_for_one():
         yield _wait_for_one()
 
 
-@coroutine
-def sleep(delay, result=None, *, loop=None):
+@types.coroutine
+def _sleep0():
+    # the hack is needed to make a quick pass
+    # of task step iteration when delay == 0
+    # Task._step has an optimization for bare yield
+    yield
```

Comment:
Please rewrite to:

```python
"""Skip one event loop run cycle.

This is a private helper for 'asyncio.sleep()', used
when the 'delay' is set to 0.  It uses a bare 'yield'
expression (which Task._step knows how to handle)
instead of creating a Future object.
"""
```